sdk/src/types/block/rand/output/feature.rs

```diff
@@ -36,6 +38,11 @@ pub fn rand_tag_feature() -> TagFeature {
     TagFeature::new(bytes).unwrap()
 }
 
+/// Generates a random [`BlockIssuerFeature`].
+pub fn rand_block_issuer_feature() -> BlockIssuerFeature {
+    BlockIssuerFeature::new(rand_number::<u64>(), vec![rand_public_key()]).unwrap()
```

Comment:
Maybe add `rand_public_keys()` similar to `rand_block_ids()` to generate public keys in the range of the allowed values and not always only a single one